Update roadmap.md (#15376)
unlock-protocol/unlock: docs/docs/governance/roadmap.md

- An [unlock-js](https://www.npmjs.com/package/@unlock-protocol/unlock-js) library that developers can use to interact with the contracts,
- A [paywall application](https://paywall.unlock-protocol.com) that can be embedded on any website or application to limit access to members and then lets users purchase keys to unlock content,
- An [unlock-app](https://app.unlock-protocol.com/locks) application that lets creators deploy their lock and view their members, as well as lets consumers view the keys they purchased and interact with them (keychain). The Unlock app also provides user accounts that enable people to create an account with their email and password and then purchase keys with their credit cards through Unlock Inc,

> - The [Unlock DAO Tokens](/governance/unlock-dao-tokens), aka UP, a governance token for the Unlock ecosystem, is used to share ownership, as well as incentivize the use of the protocol.

Author: ```suggestion
- The [Unlock Protocol Tokens](/governance/unlock-dao-tokens), aka UP, governance tokens for the Unlock ecosystem, are used to share ownership, as well as incentivize the use of the protocol.
```

> - The [Unlock Protocol Tokens](/governance/unlock-dao-tokens), aka UP, governance tokens for the Unlock ecosystem, are used to share ownership, as well as incentivize the use of the protocol.

Author: ```suggestion
- [Unlock Protocol Tokens](/governance/unlock-dao-tokens), aka UP, are governance tokens for the Unlock ecosystem, and are used to share ownership, as well as incentivize the use of the protocol.
```
